src/im_tables/utils.cljs

```diff
@@ -0,0 +1,66 @@
+(ns im-tables.utils
+  (:require [oops.core :refer [ocall]]
+            [inflections.core :refer [plural]]
+            [imcljs.path :as path]
+            [clojure.string :as string]
+            [goog.i18n.NumberFormat.Format])
+  (:import [goog.i18n NumberFormat]
+           [goog.i18n.NumberFormat Format]))
+
+(let [numformat (NumberFormat. Format/DECIMAL)]
+  (defn hr-number
```

Comment:
I keep on reading this as a human resources number 😆 could we expand it to human-readable-number?

Reply:
I think that's way too long. I'm open to other suggestions though. [=

Reply:
human-number? I'm 100% in favour of long readable names over short hard to guess names.

Reply:
What about `pretty-number`?

Reply:
I like it! 🌷